libs/feature-flags/src/lib/features.ts

```diff
@@ -15,4 +15,6 @@ export enum Features {
   // Application visibility flags
   exampleApplication = 'isExampleApplicationEnabled',
   accidentNotification = 'isAccidentNotificationEnabled',
+  drivingInstructorRegistrations = 'isDrivingInstructorRegistrationsEnabled',
+  drivingSchoolConfirmation = 'isDrivingSchoolConfirmationsEnabled',
```

Comment:
This is a complete nitpick: Either add the plural 's' to the property name, or remove it from the configcat key
```suggestion
  drivingSchoolConfirmations = 'isDrivingSchoolConfirmationsEnabled',
```
or
```suggestion
  drivingSchoolConfirmation = 'isDrivingSchoolConfirmationEnabled',
```

Reply:
will do! 😄